go-runtime/schema/common/fact.go

```diff
@@ -41,11 +40,6 @@ type SchemaFactValue interface {
 // ExtractedDecl is a fact for associating an object with an extracted schema decl.
 type ExtractedDecl struct {
 	Decl schema.Decl
```

Comment:
Is this type still useful now that it's just `Decl`?

Reply:
yeah, it needs to be wrapped in a "fact"